backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/UpdateBehaviorGroupRequest.java

```diff
@@ -15,6 +16,7 @@ public class UpdateBehaviorGroupRequest {
     /**
      * If not null, changes the display name of the behavior group to this value.
      */
+    @Size(max = 150, message = "the display name cannot exceed {max} characters")
```

Comment:
We should probably make the display name `@NotBlank` here to be consistent with `CreateBehaviorGroupRequest`.

Reply:
I wasn't sure whether to do it on this one or to open a new PR, but since you commented this, I have included that check too :+1: